src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -602,6 +604,27 @@
             "PeriodicEnergyImported",
             "PeriodicEnergyExported"
         ],
+        "Electrical Power Measurement": [
+            "PowerMode",
```

Comment:
FeatureMap too?  Same for the other .json file.

Reply:
This is a list of attributes that are:

1. Not of type list (those are handled automatically)
2. Implemented via AttributeAccessInterface, not stored in the attribute store.

So the only clusters that would list FeatureMap here are the ones that implement it via AttributeAccessInterface.  But this cluster does, right?

Reply:
Fixed.